### PR TITLE
fix: Update jaxb-runtime

### PR DESCRIPTION
**Pull request by caalador**
Update jaxb-runtime to a version
compatible with jdk17

Fixes vaadin/spring#981

### CODE REVIEW

**Review by Artur- — APPROVED** [review]
Looks fine but are these now the same deps as used elsewhere in the project?

**Comment by caalador** [comment]
Yes the versions match inside the project.

File: flow-tests/vaadin-spring-tests/pom.xml [hunk 1]
```diff
@@ -502,7 +502,7 @@
         <profile>
             <id>java 11</id>
             <activation>
-                <jdk>11</jdk>
+                <jdk>[9,)</jdk>
```
Comment (Artur-) [inline flow-tests/vaadin-spring-tests/pom.xml:505]: This profile could be removed next.. it will always be active as we only support Java 11+.

Comment (caalador) [reply flow-tests/vaadin-spring-tests/pom.xml]: done

**Review by Artur- — APPROVED** [review]

**Review by Artur- — APPROVED** [review]

**Pull request merged**